### ISSUE TITLE
Interim prior should be varied

### ISSUE THREAD

**Issue by aimalz** [aimalz/prob-z#27]
I need to run code on a more obviously inappropriate interim prior rather than the multivariate Gaussian centered with a mean of a flat distribution normalized to the expected survey size.  One variant should be an interim prior derived from poor support in redshift-color space.

**Comment by davidwhogg** [comment]
remind me -- this is a prior on what?

**Comment by aimalz** [comment]
This was one of your suggestions for something necessary to wrap up the paper.  I thought you meant the prior I used in the calculation of the posterior, and it makes sense to vary that to a prior that's more obviously inappropriate for the data set.  This would be analogous to doing inference on a test set of photometric data using a prior based on a spectroscopically confirmed training set with poor support in the parts of parameter space where the test set is.

**Comment by davidwhogg** [comment]
I still don't know what you mean.   This is a prior on what?  What is the _argument_ of the prior?

**Comment by aimalz** [comment]
Ah, sorry, the prior is over theta, the parameters specifying N(z).  This is the prior that goes into the full log posterior calculation.

**Comment by davidwhogg** [comment]
I don't understand how or why this would be called an "interim prior"?  Can you explain?

**Comment by aimalz** [comment]
My understanding was that it's used as a prior before there's evidence from the data to support that prior.  We have to specify some distribution to govern the values of the parameters but can't actually have a good idea of what that should be except by using a posterior from a previous calculation, hence the "interim" prior for the time before we have an idea of what an appropriate prior distribution might be.  This prior is used in the calculation of the posterior and to generate proposal values of the parameters at each iteration of MCMC.  If this wasn't what you had in mind, please clarify what you meant.  The only other thing I could think of was for that to be the distribution from which the initial values of the walkers are sampled, but in my case that's the same as the prior used in calculations of the posterior in MCMC and I had not considered making these distinct distributions.

**Comment by davidwhogg** [comment]
I think there is a terminology issue.  Priors are _always_ prior to the data.  There are parameters, priors, and hyperpriors.  Interim priors are what is used in a per-object inference of z before we do the hierarchical inference.  I think for this you just mean The Prior.  Or the prior on the hyperparameters.  The amplitudes that construct the N(z) function are hyperparameters.  The redshifts are parameters.  N(z) or p(z) is the redshift distribution (not the prior).  Any p(z) used (before hierarchical inference) to make the photo-zs is the interim prior.

Can you edit this issue or close and reopen it with good terminology?

**Comment by aimalz** [comment]
I changed the title, sorry about that!  I still sometimes get lost when it comes to language.  Related: I ran some tests with varying the prior and will add those to the paper today.

**Comment by davidwhogg** [comment]
good and good.  The terminology is critical, because this stuff is confusing as all get out!
